tensorflow-frameworks/tensorflow-data/src/test/java/org/tensorflow/data/DatasetTestBase.java

```diff
@@ -0,0 +1,39 @@
+package org.tensorflow.data;
+import org.junit.Before;
+import org.tensorflow.Tensor;
+import org.tensorflow.types.TInt32;
+
+import java.nio.IntBuffer;
+
+public class DatasetTestBase {
+  int[][] testMatrix1;
+  int[][] testMatrix2;
+
+
+  @Before
+  public void setUp() {
+    testMatrix1 = new int[][]{
+        {1, 2, 3, 4, 5},
+        {2, 4, 6, 8, 10},
+        {3, 6, 8, 12, 15},
+        {4, 8, 12, 16, 20}
+    };
+
+    testMatrix2 = new int[][]{
+        {1}, {0}, {1}, {1}
+    };
```

Comment:
`testMatrix1` and `testMatrix2` can be of the type `IntNdArray` that you initialize with: `StdArrays.copyOf(new int[][]{ ... })`

Reply:
Yup, I updated these!